### ISSUE TITLE
Delete in podcast episode view fails if track was deleted from the library already

### ISSUE THREAD

**Issue by Robbt** [LibreTime/libretime#689]
**Describe the bug**
Basically because of the loose coupling of the Podcast episodes table and view and the track library if you delete a track from the library and then attempt to delete it from the Podcast it fails with the error 
` [ErrorController.php:54 - errorAction()] - An internal application error has occurred.: Exception: Could not recall file with id: 13 in /vagrant/airtime_mvc/application/models/StoredFile.php:601 Stack trace: #0 /vagrant/airtime_mvc/application/controllers/LibraryController.php(260): Application_Model_StoredFile::RecallById(13) #1 /vagrant/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): LibraryController->deleteAction()`

In addition downloading the track again might cause other errors in the UI with a stuck download indicator.

**To Reproduce**
1. Add a podcast
2. Download a episode
3. Delete the track in the Library view
4. Attempt to delete the track in the Podcast Episode view

**Expected behavior**
It would see that the file was deleted and not produce an error
**LibreTime version:**
Head with PR #659 installed

**Installation method:**
 - OS: [e.g. Ubuntu]
 - OS Version [e.g. 16.04.5 LTS (Xenial Xerus)] 
 - Method: vagrant ../install

**Comment by frecuencialibre** [comment]
would a solution be to trigger a refresh of the podcast episode table with js upon deletion of the file from the library?

is this only happening on the #659 branch? seems unrelated...

**Comment by Robbt** [comment]
Yeah it is unrelated. I think that a refresh of the podcast episode table would work, or perhaps just performing a check that the file exists before deleting it from the podcast episode table. If you delete the track while not viewing the screen it will be deleted when the podcast table is refreshed. I would consider this to be a relatively minor bug but just wanted to not it.
